public/app/plugins/panel/state-timeline/timeline.ts

```diff
@@ -406,25 +403,22 @@ export function getConfig(opts: TimelineCoreOptions) {
     }
   }
 
-  function hoverOne(cx: number, cy: number) {
+  function findCurrentlyHovered(cx: number, cy: number) {
     let found: Rect | null = null;
-
     qt.get(cx, cy, 1, 1, (o) => {
       if (pointWithin(cx, cy, o.x, o.y, o.x + o.w, o.y + o.h)) {
         found = o;
       }
     });
 
-    if (found) {
-      setHoverMark(0, found);
-    } else if (hovered[0] != null) {
-      setHoverMark(0, null);
-    }
+    return found;
   }
 
-  const doHover = mode === TimelineMode.Changes ? hoverMulti : hoverOne;
```

Comment:
this was here specifically to remove the mode comparison test during each mousemove event